lms/djangoapps/mobile_api/settings.py

```diff
@@ -0,0 +1,10 @@
+'''
+	Settings for Mobile Api
+'''
+
+FB_SETTINGS = {	'_FACEBOOK_APP_SECRET' : "6c26348ef355f53a531890e980ddc731",
```

Comment:
The intent of this was that it will be moved. 
The files that contain these settings should not be in the checked in repo anyways and I don't know what app id and app secret the final version will be using. 
I structured this so that this file can be removed and all the imports that reference this can be easily replaced with the real app details. 
Let me know if there is another way you want to go about this.